javatests/arcs/android/storage/database/AndroidSqliteDatabaseManagerTest.kt

```diff
@@ -152,4 +152,71 @@ class AndroidSqliteDatabaseManagerTest {
         // The database has NOT been reset.
         assertThat(database.get(key, DatabaseData.Entity::class, schema)).isEqualTo(entity)
     }
+
+    @Test
+    fun test_getEntitiesCount() = runBlockingTest {
+        // A manager with a larger maximum size (50 kilobytes).
+        manager =
+            AndroidSqliteDatabaseManager(ApplicationProvider.getApplicationContext(), null, 50000)
+        val onDiskDatabase = manager.getDatabase("foo", true)
+        val inMemoryDatabase = manager.getDatabase("bar", false)
+        onDiskDatabase.insertOrUpdate(key, entity)
+        inMemoryDatabase.insertOrUpdate(key, entity)
```

Comment:
maybe add more than one database of the given type (eg two ondisk databases) to ensure it is taking the sum

Reply:
Good idea. Added.